google/cloud/sql/connector/client.py

```diff
@@ -189,22 +186,15 @@ async def _get_ephemeral(
     ) -> tuple[str, datetime.datetime]:
         """Asynchronously requests an ephemeral certificate from the Cloud SQL Instance.
 
-        :type project: str
-        :param project : A string representing the name of the project.
-
-        :type instance: str
-        :param instance: A string representing the name of the instance.
-
-        :type pub_key:
-        :param str: A string representing PEM-encoded RSA public key.
-
-        :type enable_iam_auth: bool
-        :param enable_iam_auth
-            Enables automatic IAM database authentication for Postgres or MySQL
-            instances.
+        Args:
+            project (str):  A string representing the name of the project.
+            instance (str):  string representing the name of the instance.
+            pub_key (str): A string representing PEM-encoded RSA public key.
+            enable_iam_auth (bool): Enables automatic IAM database
+                 authentication for Postgres or MySQL instances.
 
-        :rtype: str
-        :returns: An ephemeral certificate from the Cloud SQL instance that allows
+        Returns:
+            An ephemeral certificate from the Cloud SQL instance that allows
             authorized connections to the instance.
```

Comment:
This return type is actually out of date, it is a tuple containing the ephemeral certificate as well as a datetime object representing the expiration time of the certificate.